media/js/firefox/hello/start.js

```diff
@@ -284,17 +320,17 @@
             return;
         }
 
-        currentParam = 'incomingConversation=' + currentValue;
+        currentParam = currentValue;
```

Comment:
Is `currentParam` still needed here? Could probably get rid of it and just use `currentValue`